spec/ruby/core/array/pack/buffer_spec.rb

```diff
@@ -43,7 +43,7 @@
 
     it 'does not keep buffer content if it is longer than offset + result' do
       n = [ 65, 66, 67 ]
-      buffer = "1234567890"
+      buffer = "1234567"
```

Comment:
Is there a reason for this spec change?

Reply:
Yes, it's to keep this string embedded on 32 bit machines

Reply:
Why does this string need to be embedded? The specs shouldn't be testing things specific to MRI implementation.

Reply:
There was a bug here unrelated to shapes (thanks for pairing with me on it ❤️ ). Addressed in #6704